### PR TITLE
Feat/cronjob async create audit

### PR DESCRIPTION
**Pull request by datphamcode295**
#### What's this PR do?

- [x] cronjob async create audit

### CODE REVIEW

File: pkg/cronjob/audit/audit.go [hunk 1]
```diff
@@ -19,10 +26,288 @@ func New(store *store.Store, repo store.DBRepo, service *service.Service, logger
...
+}
+
+func (c *cronjob) createAudit(page *notion.Page, auditCycle *model.AuditCycle) error {
+	audit, err := c.service.Notion.GetBlockChildren(page.ID)
+	if err != nil {
+		c.logger.Error(err, "failed to get audit from notion")
+		return err
+	}
+
+	if page.ID == "0fde875d-67b1-4c23-b524-4b541b301c62" || page.ID == "23d6b63b-3a2b-49fc-8af7-5ffca1379a29" {
```
Comment (thanhpn) [inline pkg/cronjob/audit/audit.go:83]: should we use constant for these id

File: pkg/model/audits.go [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+import (
+	"strings"
+	"time"
+
+	"github.com/dstotijn/go-notion"
+)
+
+type AuditType string
+
+// values for audit_status
```
Comment (thanhpn) [inline pkg/model/audits.go:12]: Are there value for audit_type ?

File: pkg/store/audit/audit.go [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+	"github.com/dwarvesf/fortress-api/pkg/model"
+)
+
+type store struct{}
+
+func New() IStore {
+	return &store{}
+}
+
+// All get all audit cycle
```
Comment (thanhpn) [inline pkg/store/audit/audit.go:15]: This is Audit model not audit cycle

File: pkg/cronjob/audit/audit.go (hunk 1 above)
Comment (datphamcode295) [reply pkg/cronjob/audit/audit.go]: updated

**Review by thanhpn — APPROVED** [review]

**Pull request merged**